coalib/bearlib/__init__.py

```diff
@@ -64,12 +64,26 @@ def deprecate_settings(**depr_args):
     >>> run(old='Hello!', new='Hello!')
     Hello!
 
+    Multiple deprecations can be provided for the same setting. Note that
+    the modifier function if required must always be given as the last element
+    of the deprecated settings tuple.
+
+    >>> @deprecate_settings(new=('old','older'))
```

Comment:
maybe alllow a dict so for every old value you can give a conversion function or just None for the `lambda x:x`